pyproject.toml

```diff
@@ -18,39 +18,43 @@ classifiers = [
         'Operating System :: MacOS :: MacOS X',
         'Operating System :: Unix',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
-numpy = "^1.19.5"
-pillow = "^8.3.2"
-smqtk-core = ">=0.18.0"
-smqtk-dataprovider = ">=0.16.0"
+python = "^3.8"
+pillow = ">=9.2.0"
+numpy = ">=1.22"
+smqtk-core = "*"
+smqtk-dataprovider = "*"
 
 
 [tool.poetry.dev-dependencies]
 # CI
-flake8 = "^3.9.0"
-flake8-mutable = "^1.2.0"
-mypy = "^0.812"
+flake8 = "*"
+flake8-mutable = "*"
+mypy = "*"
```

Comment:
For things like this I think I'm much more comfortable with the `"*"` declaration as they are more utilities that we use as opposed to software. The pinned versions in the lock file are in effect solely for reproducibility of results between developers and CI.